lib/Jsrt/JsrtDebugManager.cpp

```diff
@@ -452,11 +448,6 @@ void JsrtDebugManager::CallDebugEventCallbackForBreak(JsDiagDebugEvent debugEven
     {
         tempScriptContext->GetDebugContext()->GetProbeContainer()->AsyncDeactivate();
     }
-
-    if (Js::Configuration::Global.EnableJitInDebugMode())
-    {
-        scriptContext->GetThreadContext()->GetDebugManager()->GetDebuggingFlags()->SetForceInterpreter(false);
-    }
 }
```

Comment:
this function SetForceInterpreter, should be removed as well, I don't think anyone else is calling it other than debugger.

Reply:
Thank you @akroshg! I am going to remove it as well :)